en/7-iterate-improve-frequently.md

```diff
@@ -32,390 +15,170 @@ altLangPage: 7-effectuer-iterations-ameliorations-constamment
-- [3. Agile and user-centred process (Digital Service Standard (AU))](https://www.dta.gov.au/standard/3-agile-and-user-centred/)
-- [3. Review and improve services continually (Think - Digital Design Playbook (ISED)) (internal to Government of Canada)](http://www.gcpedia.gc.ca/wiki/DDPlayBook_Think#3._Review_and_improve_services_continually)
-- [3. Apply agile principles and be iterative. (Do - Digital Design Playbook (ISED)) (internal to Government of Canada)](http://www.gcpedia.gc.ca/wiki/DDPlayBook_Do#3._Apply_agile_principles_and_be_iterative.)
-- Agile - Small chunks - Iterate (GC EA Principles)
-- {: .dpgn-digital-architectural} Use agile ({{ site.DigitalArchitectural.en }})
-
-</section>
-</section>
-
-<section class="dpgn-section-guideline">
-
-## 7.2 Accept that change is inevitable and use adaptive strategies and tools for new development
-
-<div class="dpgn-section-intro-guideline">
-
-**[TODO: Add/revise introductory text]**
-
-</div>
-
-<section class="dpgn-section-checklist">
-
-### Checklist
-
-**[TODO: Add/revise checklist items]**
-
-</section>
-
-<section class="dpgn-section-guides">
-
-### Implementation guides
-
-**[TODO: Add/revise implementation guide items]**
-
-</section>
-
-<section class="dpgn-section-solutions">
-
-### Reusable solutions
+#### US
+- [Build the service using agile and iterative practices (Digital Services Playbook (US))](https://playbook.cio.gov/#play4)
+- [Agile Principles & Practices (18F (US))](https://pages.18f.gov/agile/index.html)
+- [Is your project using ‘agilefall’? (18F (US))](https://18f.gsa.gov/2015/12/29/is-your-project-using-agilefall/)
+- [Method Cards (18F (US))](https://methods.18f.gov/)
 
-**[TODO: Add/revise reusable solutions]**
+#### AU
+- [Agile and user-centred process (Digital Service Standard (AU))](https://www.dta.gov.au/standard/3-agile-and-user-centred/)
+- [Agile (Digital Service Standard (AU))](https://www.dta.gov.au/standard/design-guides/agile/)
+- [Service design and delivery process (Digital Service Standard (AU))](https://www.dta.gov.au/standard/service-design-and-delivery-process/)
 
+#### Other
+- [Scrum (Scrum Alliance)](https://www.scrumalliance.org/)
+- [Lean (internal to Government of Canada)](http://www.gcpedia.gc.ca/wiki/Category:Lean_process_improvement_community)
+- [GCpedia Community of Practice Agile Methods (internal to Government of Canada)](http://www.gcpedia.gc.ca/wiki/Ellen_Grove_-_Agile_Methods)
+- [CRA Agile Framework (internal to Government of Canada)](http://www.gcpedia.gc.ca/wiki/Sprint_(CRA_Agile_Framework))
+- [Manifesto for Agile Software Development (agilemanifesto.org)](http://agilemanifesto.org/)
+)
 </section>
 </section>
 
 <section class="dpgn-section-guideline">
 
-## 7.3 Embrace and react to changes in the environment and design for sustainability
+## 7.2 Start small and test designs and assumptions continually, using evidence as the basis for iteration
```

Comment:
The titles for 7.1 to 7.4 shouldn't be changed here since they come directly from the Digital Standards (same for the first four guidelines of each of the other standards). If those titles should be changed, then they should be changed in the Digital Standards first and then the playbook can be updated to reflect those changes. It is different for any guidelines beyond the first four as they aren't directly related to the Digital Standards so are more easily changed.